### PR TITLE
VTA-467: Extend odometer columns and odometer history column

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## VTA-467: Correct Odometer Date Column

Modified the CSS file, added min and max width.

[VTA-467](https://jira.dvsacloud.uk/browse/VTA-467)

## Checklist

- [X] Code has been tested manually
- [X] PR title includes the JIRA ticket number
- [X] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/main/resources/assets/stylesheets/cvs-version-1.hbs [hunk 1]
```diff
@@ -184,7 +184,8 @@ vertical-align: top
 }
 
 .header__three-columns {
-display: table
+display: table;
```
Comment (cb-cs) [inline src/main/resources/assets/stylesheets/cvs-version-1.hbs:187]: It looks like this CSS is used when generating both the fail and VTG30 certificates (not only in Pass and PassNoSeatbelts). Have you checked those scenarios to see what the certficate looks like?

File: src/main/resources/assets/stylesheets/cvs-version-1.hbs [hunk 2]
```diff
@@ -200,7 +201,8 @@ vertical-align: top
 }
 
 .header__three-columns-two {
-width: 120.3333pt;
+min-width: 120.3333pt;
```
Comment (cb-cs) [inline src/main/resources/assets/stylesheets/cvs-version-1.hbs:204]: It looks like this CSS is used when generating both the fail and VTG30 certificates (not only in Pass and PassNoSeatbelts). Have you checked those scenarios to see what the certficate looks like?

File: src/main/resources/assets/stylesheets/cvs-version-1.hbs (hunk 1 above)
Comment (Daniel-Searle) [reply src/main/resources/assets/stylesheets/cvs-version-1.hbs:187]: Yes, the VTG30 cert has it all on one line e.g. 9,999,999 kilometres

File: src/main/resources/assets/stylesheets/cvs-version-1.hbs (hunk 2 above)
Comment (Daniel-Searle) [reply src/main/resources/assets/stylesheets/cvs-version-1.hbs:204]: Yes, it has it all on one line e.g. 9,999,999 kilometres

**Review by m-coslett — APPROVED** [review]

**Review by cb-cs — APPROVED** [review]

**Comment by IuliaDVSA** [comment]
VTA-467 reviewed and approved

**Pull request merged**